client/layout/index.jsx

```diff
@@ -143,6 +143,7 @@ Layout = React.createClass( {
 		return (
 			<div className={ sectionClass }>
 				<DocumentHead />
+				<QuerySites allSites />
```

Comment:
If we're adding this to layout can we clean up other usages of `<QuerySites allSites />` in 
- [ ] client/my-sites/plugins/plugin.jsx
- [ ] client/my-sites/stats/overview.jsx
- [ ] client/signup/jetpack-connect/index.jsx
- [ ] client/signup/jetpack-connect/authorize-form.jsx